pkg/ts/core/Arc.ts

```diff
@@ -62,6 +64,7 @@ export class Arc extends EventEmitter {
   }
   addStore(storeId, store) {
     if (store && !this.stores[storeId]) {
+      console.log(this, 'addStore', storeId, store);
```

Comment:
No `console.log` in commits, is usually the rule. `logFactory` log functions are usually ok, because they are (supposed to be) easy to squelch.  All is debatable, there is no codified style guide (yet).

Reply:
*nod* sorry just missed it when debugging